### PR TITLE
on/hw2

### PR DESCRIPTION
**Pull request by lolol101**

### CODE REVIEW

**Comment by yana-tanch** [comment]
- Архитектура сети 
1. Сеть сначала увеличивает размерность изображения в 2 раза, потом возвращает обратно, что неэффективно
2. Правильный порядок (ReLU → Dropout)
(ReLU обнуляет отрицательные значения, Dropout случайно отключает 50% уже активированных нейронов, иначе получается, что Dropout сразу отключает 50% нейронов до того, как ReLU применится)

- Обучение модели 
1. В валидации два раза model.eval()

- Тестирование

1. print(len(test_loader) * batch_size)  
Тут не получится размер тестового датасета, если в этом цель 

2. В отдельной ячейке текстовой нужно указать значение метрики качества на тесте 

3. На тесте с полносвязием нужно выбивать точность ~60% и более, когда сдавали данную домашку, было много сложностей с достижением нужной точности и я выкладывала в чате ноутбук с baseline, сюда дублирую на всякий случай:
https://drive.google.com/file/d/1tL5ih3F1ikdj0m69Dp-vXF9M-fTqF6E5/view?usp=sharing
От сюда можно подчеркнуть основные идеи, как выбить хорошую точность

**Review by yana-tanch — APPROVED** [review]
